ale_linters/purescript/pure_ls.vim

```diff
@@ -0,0 +1,38 @@
+" Author: David Komer <david.komer@gmail.com> 
+" Description: Integrate ALE with purescript-language-server.
+
+call ale#Set('pure_ls_executable', 'purescript-language-server')
```

Comment:
The options should be named `ale_<language>_<linter>_<option>`, so you can write `purescript_pure_ls_...` here. `purels` might be better.